File: dotnet/01-commands/tests/SumOfDigitsKata/NumberTestTemplate2.cs
using Xunit;

namespace SumOfDigitsKata
{
    public class NumberTestTemplate2
    {
        [Theory]
        [InlineData(0, 0)]
        [InlineData(10, 1)]
        public void DigitalRoot_ShouldReturnExpectedResult(long num, int expectedRes)
        {
            Number number = new Number(num);

            long res = number.DigitalRoot();

            Assert.Equal(expectedRes, res);
        }
    }
}

File: java/01-commands/tests/sourcecode/src/test/java/com/training/app/SumOfDigitsKata/NumberTestTemplate2.java
package com.training.app.SumOfDigitsKata;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class NumberTestTemplate2 {

    @Test
    public void digitalRoot_ShouldReturnExpectedResult() {
        long num = 0;
        int expectedRes = 0;

        Number number = new Number(num);

        long res = number.digitalRoot();

        assertEquals(expectedRes, res);
    }
}
